collectors/ebpf.plugin/README.md

```diff
@@ -39,787 +35,826 @@ The following two features from the Linux kernel are used by Netdata to run eBPF
+ have identical copies in secondary storage (e.g. hard disk drive or solid-state drive), discarding and reusing their space
+ is much quicker than paging out application memory, and is often preferred over flushing the dirty pages into secondary storage
+ and reusing their space.
 
-- [`mount`](https://man7.org/linux/man-pages/man2/mount.2.html): mount filesystem on host.
-- [`umount`](https://man7.org/linux/man-pages/man2/umount.2.html): umount filesystem on host.
+On `cachestat_dirties` Netdata demonstrates the number of pages that were modified. This chart shows the number of calls
+to the function `mark_buffer_dirty`.
 
-### Networking Stack
+#### Page cache ratio
 
-Netdata monitors socket bandwidth attaching `kprobes` for internal functions.
+When the processor needs to read or write in a specific memory address, it checks for a corresponding entry in the page cache.
+If the processor hits a page cache (`page cache hit`), it reads the entry from the cache. If there is no entry (`page cache miss`),
+ the kernel allocates a new entry and copies data from the disk. Netdata calculates the percentage of accessed files that are cached on
+ memory. The ratio is calculated counting the accessed cached pages
+ (without counting [dirty pages](#dirty-pages) and pages added because of read misses) divided by total access without dirty pages.
 
-#### TCP functions
+\_\_________<ins>Number of accessed cached pages</ins>__________
 
-This chart demonstrates calls to functions `tcp_sendmsg`, `tcp_cleanup_rbuf`, and `tcp_close`; these functions are used
-to send & receive data and to close connections when `TCP` protocol is used.
+Number of total accessed pages - dirty pages - missed pages
 
-#### TCP bandwidth
+The chart `cachestat_ratio` shows how processes are accessing page cache. In a normal scenario, we expect values around
+100%, which means that the majority of the work on the machine is processed in memory. To calculate the ratio, Netdata
+attaches `kprobes` for kernel functions:
 
-Like the previous chart, this one also monitors `tcp_sendmsg` and `tcp_cleanup_rbuf`, but instead of showing the number
-of calls, it demonstrates the number of bytes sent and received.
+- `add_to_page_cache_lru`: Page addition.
+- `mark_page_accessed`: Access to cache.
+- `account_page_dirtied`: Dirty (modified) pages.
+- `mark_buffer_dirty`: Writes to page cache.
 
-#### TCP retransmit
+#### Page cache misses
 
-This chart demonstrates calls to function `tcp_retransmit` that is responsible for executing TCP retransmission when the
-receiver did not return the packet during the expected time.
+A page cache miss means that a page was not inside memory when the process tried to access it. This chart shows the
+result of the difference for calls between functions `add_to_page_cache_lru` and `account_page_dirtied`.
 
-#### UDP functions
+#### File sync
```

Comment:
Synchronization and Page Cache monitors data in memory, but they are provided by different threads, should we use different levels for them?

Reply:
Yes, sounds good. Just to clarify a bit: By which criteria do we sort this big list?

Once I know the criteria, I can take another look and find a logical layout for the whole document. 

Are we sorting by thread, and the threads appear in alphabetical order?

Reply:
We were sorting data according the dashboard visualization. Both `Synchronization (eBPF)`  and `Page Cache(eBPF)` are inside `Memory` submenu.